Makefile.am

```diff
@@ -25,6 +25,7 @@ SUBDIRS += po
 %.service: %.service.in config.log
```

Comment:
The latter paragraph in the commit message does not seem to apply to the code at hand anymore.

Reply:
Yeah, the paragraph I refer to was
```
Also, make sure to run with XDG_DATA_DIRS set. Otherwise we can see in
the log that update-mime-database will complain about it not being set
when installing an application using the system helper.
```

As far as I can see, the helper already runs with `XDG_DATA_DIRS` set; this patch only adds a new path to it.

Reply:
I edited the commit message. It looks like it's been inaccurate since at least the 3.0.4 release so maybe it was always that way.